static/src/javascripts-legacy/projects/common/modules/atoms/youtube.js

```diff
@@ -71,7 +73,33 @@ define([
         }
     }
 
+
+
+    function shouldAutoplay(){
+
+        function isMobile() {
+            const isIOS = () => /(iPad|iPhone|iPod touch)/i.test(navigator.userAgent);
```

Comment:
this shouldn't be es6 no, i'm surprised it linted? it won't be transpiled anyway. 

you probably also want to use https://github.com/guardian/frontend/blob/master/static/src/javascripts-legacy/projects/common/utils/detect.js#L164